misc/testnnp/Dockerfile

```diff
@@ -11,7 +11,7 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-FROM gcc:4.9
+FROM gcc:6.4
```

Comment:
Can we just use the latest here, in case we will need to update again in the future?

Reply:
I don't want to use `latest` as it makes it hard to audit and track changes